crates/wit-component/src/printing.rs

```diff
@@ -51,61 +51,65 @@ impl WitPrinter {
     }
 
     /// Print a set of one or more WIT packages into a string.
-    pub fn print(
-        &mut self,
-        resolve: &Resolve,
-        pkg_ids: &[PackageId],
-        force_print_package_in_curlies: bool,
-    ) -> Result<String> {
-        let print_package_in_curlies = force_print_package_in_curlies || pkg_ids.len() > 1;
-        for (i, pkg_id) in pkg_ids.into_iter().enumerate() {
+    pub fn print(&mut self, resolve: &Resolve, pkg_ids: &[PackageId]) -> Result<String> {
```

Comment:
Could this change to taking a single `PackageId` instead of a list? I think it's a bit confusing to have the "main" one be at the end and it would help emphasize that this only prints a single package at a time. 

I'm imagining that internally it would use `LiveTypes` to calculate the set of packages that this package depends on and then all of those packages would be printed in curly braces at the end. That way the semantic meaning of the output is the same but this is a way to print a whole package as a self-contained entity.

As an optional part the "print all deps too" logic could be controlled by a `bool` parameter, but that's ok to defer to later.